apps/dashboard/src/components/primitives/scroll-area.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as ScrollAreaPrimitive from '@radix-ui/react-scroll-area';
+
+import { cn } from '@/utils/ui';
+
+const ScrollArea = React.forwardRef<
```

Comment:
Added a new shadcn scrollable area for cross-browser scrollable elements (vertical and horizontal)